047-TrafficControlWithDapr/Coach/Solution-07.md

```diff
@@ -47,23 +45,23 @@ First, you'll add a secrets management JSON configuration file to the solution:
       type: secretstores.local.file
       version: v1
       metadata:
-      - name: secretsFile
-        value: ../dapr/components/secrets.json
-      - name: nestedSeparator
-        value: "."
+        - name: secretsFile
+          value: ../dapr/components/secrets.json
+        - name: nestedSeparator
+          value: "-"
     scopes:
-      - finecollectionservice   
+      - fine-collection-service
     ```
 
-    *Note the `type` element: The `local.file` secret store is specified. This file-based local secret store component is **only** for development or testing purposes. It's not suitable for production!*
+    _Note the `type` element: The `local.file` secret store is specified. This file-based local secret store component is **only** for development or testing purposes. It's not suitable for production!_
 
-    *Understand that if you specify a relative path to the `secretsFile` (as is the case here), this path must be specified relative to the folder where you start your service using the Dapr CLI. Because you start the services from their project folders, the relative path to the components folder is always `../dapr/components`.*
+    _Understand that if you specify a relative path to the `secretsFile` (as is the case here), this path must be specified relative to the folder where you start your service using the Dapr CLI. Because you start the services from their project folders, the relative path to the components folder is always `../dapr/components`._
 
-The `nestedSeparator` in the `metadata` section specifies the character that Dapr will use when it flattens the secret file's hierarchy. Each secret will be uniquely identifiable by one key. In this case, you're using the period (`.`) as that character. The convention means that secrets from the `secrets.json` file will be identified by the following keys:
+The `nestedSeparator` in the `metadata` section specifies the character that Dapr will use when it flattens the secret file's hierarchy. Each secret will be uniquely identifiable by one key. In this case, you're using the period (`-`) as that character. The convention means that secrets from the `secrets.json` file will be identified by the following keys:
```

Comment:
Do you mean "you're using the hyphen as that character"?

Reply:
yes, that should be changed

Reply:
fixed